test/unit/spec/LoginRouter_spec.js

```diff
@@ -538,23 +538,26 @@ function (Okta, Q, Backbone, XDomain, SharedUtil, CryptoUtil, CookieUtil, OktaAu
     Expect.describe('OIDC - okta is the idp and oauth2 is enabled', function () {
 
       function expectAuthorizeUrl(url, options) {
-          var expectedUrl = 'https://foo.com/oauth2/v1/authorize?' +
-            'client_id=someClientId&' +
-            'redirect_uri=https%3A%2F%2F0.0.0.0%3A9999&' +
-            'response_type=' + options.responseType + '&' +
-            'response_mode=' + options.responseMode + '&' +
-            'state=' + OIDC_STATE + '&' +
-            'nonce=' + OIDC_NONCE + '&';
-          if (options.display) {
-            expectedUrl += 'display=' + options.display + '&';
-          }
-          if (options.prompt) {
-            expectedUrl += 'prompt=' + options.prompt + '&';
-          }
-          expectedUrl += '' +
-            'sessionToken=THE_SESSION_TOKEN&' +
-            'scope=openid%20email';
-          expect(url).toBe(expectedUrl);
+        var authorizeUrl = options.authorizeUrl || 'https://foo.com/oauth2/v1/authorize';
```

Comment:
It might be nice to setup an e2e test for this as well; I remember you had mentioned looking into it - was there a reason to not also add that test?